pkg/controllers/logs/controller_test.go

```diff
@@ -85,21 +83,10 @@ func Test_ControllerFilterLogs(t *testing.T) {
 		var resp string
 		var expected []byte
 		var status, expectedStatus int
-		parameters := logs.Parameters{
-			Index:      "",
-			Level:      "",
-			FinishTime: "",
-			StartTime:  "",
-			Podname:    "",
-			Namespace:  "",
-		}
 		switch tt.TestName {
 		case "Filter by no parameters":
 			rr := httptest.NewRecorder()
-			gctx, _ := gin.CreateTestContext(rr)
-			body := new(bytes.Buffer)
-			json.NewEncoder(body).Encode(parameters)
-			req, err := http.NewRequestWithContext(gctx, http.MethodGet, "/logs/filter", body)
+			req, err := http.NewRequest(http.MethodGet, "/logs/filter", nil)
```

Comment:
I think you can add request parameters to the test itself, in this case, you won't need this `switch tt.TestName` structure.

Reply:
@igor-karpukhin Tried to improve the test, please check and let me know if it's okay or not.